Split-Directions.parset

```diff
@@ -54,7 +59,7 @@
 ##########################################################
 
 ## data column from Delay-Calibration
-! data_col = CORRECTED_DATA 
+! data_col = DATA 
```

Comment:
change to

! data_col = DATA   ## set to DATA_DI_CORRECTED if you applied ddf solutions in Delay-Calibration